constants.py

```diff
@@ -38,18 +38,18 @@ class Modes:
     """
     all the modes that are supported.
     """
-    MODES_LIST = ['N/A', 'CW', 'AM', 'FM', 'LSB', 'USB', 'RTTY', 'PSK', 'PSK31', 'PSK63', 'FT8', 'FT4']
+    MODES_LIST = ['N/A', 'CW', 'AM', 'FM', 'LSB', 'USB', 'SSB', 'RTTY', 'PSK', 'PSK31', 'PSK63', 'FT8', 'FT4', 'MFSK', 'NoMode', 'None']
```

Comment:
does your logging software (I think you are not using N1MM) emit a mode of 'SSB'?

Reply:
I have seen other software send SSB. I think TR4W (which I maintain) did but I fixed that issue. This is a belt-and-suspenders approach now more so just in case.